### ISSUE TITLE
Changing variable name

### ISSUE THREAD

**Issue by amberk887** [kkachmar/wait-time-app#15]
For clarity, I think we should change the variable name, activecases, to totalcases since it only increments, but does not account for when the case is no longer active. This variable is in the CheckResults.java file.

**Comment by kkachmar** [comment]
I agree I think totalCases speaks more to the prevalence of COVID-19 since the start of testing, while activeCases sounds more like the number of new cases.
